doc/retrieve-payments.md

```diff
@@ -0,0 +1,61 @@
+# Retrieving payments
+
+Use these methods to retrieve payment details from ThePay API — either a single payment by UID or multiple payments using filters and pagination.
+
+## Get a Single Payment
+
+To retrieve information about a single payment, call:
+
+```php
+$payment = $thePayClient->getPayment('49096fe3-872d-3cbe-b908-2806ae2d7c79');
+```
+
+**Parameters:**
+- string `$uid` — Unique identifier (UID) of the payment.
+
+The payment must belong to the project configured in TheConfig.
+
+**Returns:**
+
+An object describing the payment.
```

Comment:
I assume that this is necessary. That information should be part of the `getPayment` method's annotation.